{{cookiecutter.project_slug}}/tox.ini

```diff
@@ -1,30 +1,41 @@
 [tox]
-envlist = py35, py36, py37, py38, flake8
+envlist = py{36,37,38}, black{%- if cookiecutter.make_docs ==  'y' %}, docs{%- endif %}, py38-{macOS,Windows}
```

Comment:
I think this ` {{cookiecutter.project_slug}}/tox.ini` can probably be submitted upstream as well.